modules/hub/README.md

```diff
@@ -41,7 +41,9 @@ To deploy this config:
 | gke\_hub\_sa\_name | Name for the GKE Hub SA stored as a secret `creds-gcp` in the `gke-connect` namespace. | string | `"gke-hub-sa"` | no |
 | location | The location (zone or region) this cluster has been created in. | string | n/a | yes |
 | project\_id | The project in which the resource belongs. | string | n/a | yes |
+| sa\_private\_key | Private key for service account. Required only if `use_existing_sa` is set to `true`. | string | `"null"` | no |
```

Comment:
Should we call out the fact that this should be base64 encoded?

Reply:
Yes. That is a good idea.